CONTRIBUTING.md

```diff
@@ -66,6 +66,12 @@ Alternatively, you can use the Docker image to replicate the environment without
 
 #### With Docker
 
+* Copy toml file to ensure cacheability. Bash\Powershell:
+
+  ```shell
+  cp pyproject.toml pyproject.toml.mod
```

Comment:
Well... why? I mean... we're missing the `sed` part. But I don't think I get why would we even want to do this...

Reply:
Right now If somebody cloned the project and tried following Contributing with Docker instructions, they would get:

=> ERROR [6/8] COPY pyproject.toml.mod /app/pyproject.toml                                                                                                                                                                        0.0s 
------
 > [6/8] COPY pyproject.toml.mod /app/pyproject.toml:
------
failed to compute cache key: "/pyproject.toml.mod" not found: not found

I think that We might want to adress it in some way in Contributing instructions

Reply:
Ok, that's a valid argument. Can we do the sed part as well to maintain parity?

Reply:
Good point about PowerShell comaptibility, we can issue a comment hinting at the right replacement for `sed`

Reply:
After careful consideration: let's not put there it's ok.